### PR TITLE
Add Chinese Language Model

### PR DESCRIPTION
**Pull request by irina060981**
For the issue https://github.com/alpheios-project/components/issues/843

 - added `chinese_language_model.js`
 - register constants for ChLM
 - register radical feature for ChLM
 - register ChLM in LMF
 - added unit tests 

Should be merged with
https://github.com/alpheios-project/client-adapters/pull/39

### CODE REVIEW

File: src/chinese_language_model.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        Feature.types.mandarin,
+        []
+      ],
+      [
+        Feature.types.cantonese,
+        []
+      ],
+      [
+        Feature.types.tang,
+        []
```
Comment (kirlat) [inline src/chinese_language_model.js]: I know it's very complicated and I am not even distantly familiar with Chinese, but I'm wondering it that would be the best way to classify the varieties of the language.

From the [article](https://www.vengaglobal.com/blog/simplified-traditional-chinese-mandarin-cantonese/) I understood that traditional Chinese and simplified Chinese are more of writing methods (different ways to record approximately the same words) than dialects, even though now there are some terms and concepts unique to one or the other. Mandarin and Cantonese are varieties of a spoken language. So if we group simplified and traditional Chinese together maybe we shall group Mandarin and Cantonese too (should then they be dialects)? Or maybe all four belong to the same plane? @balmas, what do you think?

Comment (balmas) [reply src/chinese_language_model.js]: so I agree let's remove Feature.types.cantonese, Feature.types.tang, Feature.types.mandarin,Feature.types.dialect.simple, and Feature.types.dialect.traditional from the data model

Comment (irina060981) [reply src/chinese_language_model.js]: oh, good catch! I am sorry - I have forgotten to remove those features!

File: src/chinese_language_model.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      ],
+      [
+        Feature.types.pronunciation,
+        []
+      ],
+      [
+        Feature.types.dialect,
+        [
+          'simple',
```
[the remark this answers is not in the document]
Comment (irina060981) [reply src/chinese_language_model.js]: will fix it

**Comment by irina060981** [comment]
I think, I could merge it - there really few changes.

**Pull request merged**